sdk/python/lib/test/langhost/testing_with_mocks/resources.py

```diff
@@ -29,3 +29,6 @@ def do_invoke():
                       name="myvm",
                       value=pulumi.Output.secret("secret_value"))
 invoke_result = do_invoke()
+
+
```

Comment:
Uber-nit: two empty lines

Reply:
Fixed. I added a commit that moves the `pulumi.export` calls from `__main__.py` here, and fixed up the two empty lines.